runtimes/huggingface/pyproject.toml

```diff
@@ -23,8 +23,13 @@ torch = [
     {markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", version = "<2.3.0"},
     {markers = "sys_platform != 'darwin' or platform_machine != 'x86_64'", version = "<2.4.0"}
 ]
+# tensorflow 2.17.0 not compiled for MacOS
+# see: https://www.tensorflow.org/install/pip#package_location
+tensorflow = [
+    {markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", version = ">=2.12,<2.17"},
+    {markers = "sys_platform != 'darwin' or platform_machine != 'x86_64'", version = ">=2.12,<2.18"}
```

Comment:
It is probably fine, although I wondered a bit if we should be explicitly about what we include (as opposed to exclude).